source/usr/lib/mcvirt/virtual_machine/virtual_machine_config.py

```diff
@@ -66,7 +66,8 @@ def create(vm_name, available_nodes, cpu_cores, memory_allocation):
                 'network_interfaces': {},
                 'node': None,
                 'available_nodes': available_nodes,
-                'lock': LockStates.UNLOCKED.value
+                'lock': LockStates.UNLOCKED.value,
```

Comment:
Given that you've got a generic method to apply cpu-modifications, I would be good to have a 'modifications' part of the config:

    {
        ...
        modifications: {
            'windows_cpu': False
        }
    }